doc/user-guide/profiling/index.md

```diff
@@ -1,7 +1,14 @@
+---
+html_meta:
+  "description lang=en": ""
```

Comment:
add description here, you can use the same description as in the github repo: "a toolbox for..."